gui/widgets/tab.cpp

```diff
@@ -221,7 +221,8 @@ void TabWidget::handleMouseDown(int x, int y, int button, int clickCount) {
 }
 
 void TabWidget::handleMouseMoved(int x, int y, int button) {
-	assert(y < _tabHeight);
+	if (y < _tabHeight)
```

Comment:
This should be `if (y >= _tabHeight)`.
Also it would be good to check that y is positive. So change this to `if (y < 0 || y >= _tabHeight)`.